WordPress/Sniffs/XSS/EscapeOutputSniff.php

```diff
@@ -441,4 +440,18 @@ protected function mergeFunctionLists() {
 		}
 	}
 
+	/**
+	 * Checks whether short_open_tag is enabled.
+	 *
+	 * Since PHP 5.4, <?= is always available.
+	 *
+	 * @return bool False if short_open_tag is not enabled, true otherwise
+	 */
+	public function is_short_open_tag_enabled() {
+		if ( true === version_compare(phpversion(), '5.4', '<' ) && false === (bool) ini_get( 'short_open_tag' ) ) {
```

Comment:
Please use the `PHP_VERSION_ID` constant instead for the comparison (which is backfilled in PHPCS).
`PHP_VERSION_ID < 504000` is two less functions calls for the same result.